PC/layout/support/constants.py

```diff
@@ -16,7 +16,7 @@ def _unpack_hexversion():
         hexversion = int(os.getenv("PYTHON_HEXVERSION"), 16)
     except (TypeError, ValueError):
         hexversion = sys.hexversion
-    return struct.pack(">i", sys.hexversion)
```

Comment:
could we create patch files for all these changes like we have in icu? otherwise it's going to be a pain to reapply them when we update python

Reply:
If we are, I want to make an executive decision to use tooling like `quilt` for applying the patches - it's way less fragile than the NIH solution in ICU

Reply:
In this case, of course, updating to 3.11+, 3.10.3+, or 3.9.11+ means we don't need this change any more. We're using 3.9.2 because that's what EMSDK uses (even in git main)

Reply:
Let's use your tooling solution in ICU as well ;-).

Reply:
That's tooling we've taken from https://github.com/microsoft/icu so if we want to converge on their fork again later, we'd need their buy-in to make structural changes like that